src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -479,7 +479,7 @@
   - name: --enable-keda
     type: bool
     short-summary: Enable KEDA workload auto-scaler.
-  - name: --enable-azure-monitor-metrics
+  - name: --enable-azuremonitormetrics
```

Comment:
It seems that the behavior of core CLI and CLI extension is inconsistent

In core CLI, it is `--enable-azure-monitor-metrics`
https://github.com/Azure/azure-cli/blob/d97a64510838aedb921ffbb285c2c38a63c66888/src/azure-cli/azure/cli/command_modules/acs/_params.py#L285

In CLI extension, it is `--enable-azuremonitormetrics`
https://github.com/Azure/azure-cli-extensions/blob/6229e121f3bba38a4aa52c7cf30968f91e479b79/src/aks-preview/azext_aks_preview/_params.py#L437

You need to ensure that the parameter design from core CLI and CLI extension is consistent, otherwise it may easily confuse the customers.

+ @FumingZhang Please be aware of it~

Reply:
Yes, it is recommended to hide the previously unreasonable parameter names in aks-preview first

Reply:
Thanks @FumingZhang and @zhoxing-ms . Would you be creating another PR for the fixing the `aks-preview` extension ?

Reply:
Yes, created this [PR #6379](https://github.com/Azure/azure-cli-extensions/pull/6379) in aks-preview.